docs/developer-guide/syntax-reference.rst

```diff
@@ -1512,7 +1512,11 @@ Scalar functions
 | ARRAYCONTAINS          |  ``ARRAYCONTAINS('[1, 2, 3]', 3)``                                        | Given JSON or AVRO array checks if a search       |
 |                        |                                                                           | value contains in it                              |
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
-| CEIL                   |  ``CEIL(col1)``                                                           | The ceiling of a value.                           |
+| CEIL                   |  ``CEIL(col1)`` or   ``CEIL(col1, precision)``                            | The ceiling of a value.                           |
+|                        |                                                                           |                                                   |
+|                        |                                                                           | precision is a number indicating the number of    |
```

Comment:
"...number indicating the number..." maybe we can reword this?

How about: "precision is an INT indicating the number of decimal places to round up to."

Reply:
Isn't `precision` confusing with the new DECIMAL type precision? DECIMAL has (precision, scale), and scale is the number of digits after the decimal point. Should we use a similar parameter name?

Reply:
scale works for me